protoc-gen-kotlin/lib/src/jvmTest/kotlin/CodeGeneratorTest.kt

```diff
@@ -0,0 +1,58 @@
+import com.tschuchort.compiletesting.KotlinCompilation
+import com.tschuchort.compiletesting.KotlinCompilation.ExitCode
+import com.tschuchort.compiletesting.SourceFile
+import pbandk.decodeFromByteArray
+import pbandk.gen.pb.CodeGeneratorRequest
+import pbandk.gen.runGenerator
+import pbandk.wkt.FileDescriptorSet
+import java.io.File
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+
+class CodeGeneratorTest {
+    private val descriptorSetOutput = File("build/tmp/generateTestProto/fileDescriptor.protoset")
+    private val fileDescriptorSet by lazy {
+        check(descriptorSetOutput.exists()) {
+            "${descriptorSetOutput.absolutePath} does not exist, make sure it is generated via :generateTestProto"
+        }
+        FileDescriptorSet.decodeFromByteArray(descriptorSetOutput.inputStream().readAllBytes()).file
```

Comment:
You should be able to use `decodeFromStream(descriptorSetOutput.inputStream())` here